superset/views/sql_lab.py

```diff
@@ -255,7 +256,7 @@ def delete_query(  # pylint: disable=no-self-use
 
         db.session.query(Query).filter_by(
             client_id=client_id,
-            user_id=g.user.get_id(),
+            user_id=get_user_id(),
```

Comment:
Mixed types, though likely benign from the database's perspective—at least for MySQL.